cmd/backup-manager/app/export/export.go

```diff
@@ -41,8 +42,14 @@ func (bo *Options) getBackupFullPath() string {
 }
 
 func (bo *Options) getBackupRelativePath() string {
+	var backupRelativePath string
 	backupName := fmt.Sprintf("backup-%s", time.Now().UTC().Format(time.RFC3339))
-	return fmt.Sprintf("%s/%s", bo.Bucket, backupName)
+	if len(bo.Prefix) == 0 {
+		backupRelativePath = fmt.Sprintf("%s/%s", bo.Bucket, backupName)
+	} else {
+		backupRelativePath = fmt.Sprintf("%s/%s/%s", bo.Bucket, bo.Prefix, backupName)
```

Comment:
What if the prefix is "/", "/a/b//" or something similar?

Reply:
If there is a problem with the prefix format, the backup will fail, just treat it as a normal failure

Reply:
Can we trim the "/" with strings.Trim so that those cases can be avoided, otherwise, we need to make it clear in the doc for how to configure the prefix.